Tools/LambdaTestTool/src/Amazon.Lambda.TestTool.BlazorTester/Amazon.Lambda.TestTool.BlazorTester70-pack.csproj

```diff
@@ -0,0 +1,33 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <Import Project="..\..\..\..\buildtools\common.props" />
+  
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <Description>A tool to help debug and test your .NET 7.0 AWS Lambda functions locally.</Description>
+    <VersionPrefix>0.12.6</VersionPrefix>
+    <Product>AWS .NET Lambda Test Tool</Product>
+    <Copyright>Apache 2</Copyright>
+    <PackageTags>AWS;Amazon;Lambda</PackageTags>
+    <TargetFramework>net7.0</TargetFramework>
+    <ToolCommandName>dotnet-lambda-test-tool-7.0</ToolCommandName>	
+    <IsPackable>true</IsPackable>
+    <PackAsTool>true</PackAsTool>
+    <PackageId>Amazon.Lambda.TestTool-7.0</PackageId>
+	<AssemblyName>Amazon.Lambda.TestTool.BlazorTester</AssemblyName>
+	<RootNamespace>Amazon.Lambda.TestTool.BlazorTester</RootNamespace>	
+	<GenerateEmbeddedFilesManifest>true</GenerateEmbeddedFilesManifest>
+	<StaticWebAssetsEnabled>false</StaticWebAssetsEnabled>
+  </PropertyGroup>
+
+  <ItemGroup>
+      <PackageReference Include="Blazored.Modal" Version="3.1.2" />
+	  <PackageReference Include="Microsoft.Extensions.FileProviders.Embedded" Version="7.0.0-rc.2.22476.2" /> 
```

Comment:
same question here on exact .NET 7 version

Reply:
Because there is no 7 version without a suffix like `-rc2` wildcarding the version doesn't work. We will just have to do a future release switching to the GA version.